LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Cloud4RPi
+Copyright (c) 2017-2019 Cloud4RPi
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
This sentence is a bit obscure. Can you explain its function?

Reply:
That's the MIT license text, no need to correct it.